cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java

```diff
@@ -9,9 +9,9 @@
  */
 public class DebugUtils {
 
-    private static final String DEBUG_ARGS_JAVA8 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
+    private static final String DEBUG_ARGS_JAVA11 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
     private static final String JAVA_VERSION_PROP = "java.version";
-    private static final String COMPATIBLE_JRE_VERSION = "1.8";
+    private static final String COMPATIBLE_JRE_VERSION = "11";
```

Comment:
Seems like debugging will only support in Java 11. Don't we have to support both Java 8 and Java 11?

Reply:
@praveennadarajah If I'm not mistaken, our ballerina runtime won't be backward compatible with jdk8, so thought of limiting the debugger compatibility only to jdk11. @keizer619 any thoughts?

Reply:
Yes runtime will not support for 1.8